packages/client/components/GitLabScopingSearchResults.tsx

```diff
@@ -0,0 +1,242 @@
+// import useAtmosphere from '../hooks/useAtmosphere'
+// import useGetUsedServiceTaskIds from '../hooks/useGetUsedServiceTaskIds'
+// import useLoadNextOnScrollBottom from '../hooks/useLoadNextOnScrollBottom'
+// import PersistGitLabSearchQueryMutation from '../mutations/PersistGitLabSearchQueryMutation'
+// import {SprintPokerDefaults} from '../types/constEnums'
+import {GQLType} from '../types/generics'
+import getNonNullEdges from '../utils/getNonNullEdges'
+// import {gitHubQueryValidation} from '../validation/gitHubQueryValidation'
+import {GitLabScopingSearchResultsPaginationQuery} from '../__generated__/GitLabScopingSearchResultsPaginationQuery.graphql'
+import {GitLabScopingSearchResultsQuery} from '../__generated__/GitLabScopingSearchResultsQuery.graphql'
+// import {GitLabScopingSearchResults_meeting$key} from '../__generated__/GitLabScopingSearchResults_meeting.graphql'
+import {GitLabScopingSearchResults_query$key} from '../__generated__/GitLabScopingSearchResults_query.graphql'
+// import Ellipsis from './Ellipsis/Ellipsis'
+// import GitLabScopingSearchResultItem from './GitLabScopingSearchResultItem'
+// import GitLabScopingSelectAllIssues from './GitLabScopingSelectAllIssues'
+// import IntegrationScopingNoResults from './IntegrationScopingNoResults'
+// import NewGitLabIssueInput from './NewGitLabIssueInput'
+// import NewIntegrationRecordButton from './NewIntegrationRecordButton'
+
+const ResultScroller = styled('div')({
+  overflow: 'auto'
+})
+
+// const LoadingNext = styled('div')({
+//   display: 'flex',
+//   height: 32,
+//   fontSize: 24,
+//   justifyContent: 'center',
+//   width: '100%'
+// })
+
+interface Props {
+  queryRef: PreloadedQuery<GitLabScopingSearchResultsQuery>
+  // meetingRef: GitLabScopingSearchResults_meeting$key
+}
+
+const GitLabScopingSearchResults = (props: Props) => {
+  const {queryRef} = props
+  const query = usePreloadedQuery(
+    graphql`
+      query GitLabScopingSearchResultsQuery($teamId: ID!) {
+        ...GitLabScopingSearchResults_query
+        viewer {
+          id
+          # ...NewGitLabIssueInput_viewer
+          # teamMember(teamId: $teamId) {
+          #   suggestedIntegrations {
+          #     items {
+          #       # ... on SuggestedIntegrationGitLab {
+          #       #   id
+          #       #   nameWithOwner
+          #       # }
+          #     }
+          #   }
+          #   integrations {
+          #     gitlab {
+          #       gitlabSearchQueries {
+          #         queryString
+          #       }
+          #     }
+          #   }
+          # }
+        }
+      }
+    `,
+    queryRef,
+    {UNSTABLE_renderPolicy: 'full'}
+  )
+
+  const paginationRes = usePaginationFragment<
+    GitLabScopingSearchResultsPaginationQuery,
+    GitLabScopingSearchResults_query$key
+  >(
+    graphql`
+      fragment GitLabScopingSearchResults_query on Query
+        @argumentDefinitions(cursor: {type: "String"}, count: {type: "Int", defaultValue: 50})
```

Comment:
For now, this is grabbing the first 50 projects & all of their issues. In the PR for the successor issue, I'll work on more advanced pagination to get the first 25 issues